external/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisInputDStream.scala

```diff
@@ -100,6 +103,7 @@ object KinesisInputDStream {
     private var endpointUrl: Option[String] = None
     private var regionName: Option[String] = None
     private var initialPositionInStream: Option[InitialPositionInStream] = None
+    private var initialPositionInStreamTimestamp: Option[Date] = None
```

Comment:
Ah alright, so you're asking to get another `initialPositionInStreamTimestamp`. Thats similar to the `withInitialPositionAtTimestamp`.  Can rename that to suit this purpose.

Another question, The InitialPosition gets passed to the KinesisReceiver. I was passing a timestamp along with the Initial position at the moment. Are we planning to pass the `KinesisClientLibConfiguration` to the `KinesisReceiver` now ?